modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/action/helper/SaveFormInstanceMVCCommandHelper.java

```diff
@@ -364,6 +371,48 @@ private URI _getURI(String uriString) {
 		}
 	}
 
+	private void _validateExpirationDate(DDMFormValues ddmFormValues)
+		throws Exception {
+
+		if (!_ffSubmissionsSettingsConfigurationActivator.
+				expirationDateEnabled()) {
+
+			return;
+		}
+
+		Map<String, List<DDMFormFieldValue>> ddmFormFieldValuesMap =
+			ddmFormValues.getDDMFormFieldValuesMap(false);
+
+		if (!GetterUtil.getBoolean(
+				_getPropertyValue(
+					ddmFormFieldValuesMap, LocaleUtil.ROOT, "neverExpire"),
```

Comment:
@rodrigopaulino why did you decide to use the `LocaleUtil.ROOT` instead of `ddmForm.getDefaultLocale()`?

Reply:
It's because those fields are UnlocalizedValues.